cookbooks/aws-parallelcluster-test/recipes/tests.rb

```diff
@@ -101,6 +101,42 @@
     raise "node_type must be HeadNode or ComputeFleet"
   end
 end
+###################
+# Scheduler Plugin
+###################
+if node['cluster']['scheduler'] == 'plugin'
+  if node['cluster']['node_type'] == "HeadNode"
+    execute 'check artifacts are in target_source_path' do
+      command "sudo ls #{node['cluster']['scheduler_plugin']['home']} | grep develop"
+    end
+    execute 'check handler-env.json in path' do
+      command "ls #{node['cluster']['shared_dir']}/handler-env.json"
+    end
+    execute 'check get compute fleet script in path' do
+      command "ls /usr/local/bin/get-compute-fleet-status.sh"
```

Comment:
not only check script is present, but check it's executable by the the scheduler-plugin user

Reply:
Changed to "su #{node['cluster']['scheduler_plugin']['user']} -c 'if [[ ! -x '/usr/local/bin/get-compute-fleet-status.sh' ]]; then exit 1; fi;'" to check if it is executable by plugin user